packages/uxpin-merge-cli/package.json

```diff
@@ -133,7 +133,7 @@
     "sortobject": "^1.1.1",
     "source-map-support": "^0.4.18",
     "tslib": "^1.9.0",
-    "typescript": "4.4.2",
+    "typescript": "4.8.4",
```

Comment:
I am curious as to why specifically version `4.8.4`? Is it because this version was most commonly downloaded (according to npmjs stats) and the latest non-major upgraded version? I was looking through the change log but am not sure as to what fix helped with our case. 🤔

Reply:
Ipicked a popular recent version of TypeScript, I don't know what is the exact change that fixes the behavior.

With TS 4.4.2 I get undefined when I call this for the ReactNode and ReactChild:

```ts
const typeSymbol: ts.Symbol = type.symbol || type.aliasSymbol;
```

With TS 4.8.4, I got the expected type and `typeSymbol.escapedName` contains either `ReactNode` or `ReactChild`, that are valid keys in the map object we use to detect the known types.